rosbag2_converter_default_plugins/plugin_description.xml

```diff
@@ -0,0 +1,9 @@
+<library path="rosbag2_converter_default_plugins">
+<class
+        name="cdr_converter"
+        type="CdrConverterPlugin"
+        base_class_type="rosbag2::FormatConverterInterface"
+>
+  <description>This is a converter plugin for cdr format.</description>
+</class>
+</library>
```

Comment:
```suggestion
</library>

```

Not sure if this helps, but github _complains_ that there is no new line for this file.

Reply:
I guess it makes sense to fix that - I expect the xml linter to find this problem later on once it'll be enabled.